### ISSUE TITLE
Display of Gender icons distorted

### ISSUE THREAD

**Issue by mferrare** [Digital-Law-Lab/docassemble-CLCSA01ServiceDirectory#27]
Gender selection looks like this:
![image](https://user-images.githubusercontent.com/582798/230831681-835f86e8-fd45-4fe7-98ad-9cbf2a84fa1b.png)

_Originally posted by @mferrare in https://github.com/Digital-Law-Lab/docassemble-clcsa-01-service-directory/issues/24#issuecomment-1501409099_

**Comment by mferrare** [comment]
I converted both PNGs to SVG and then had to modify the SVGs according to what ChatGPT told me as below:

<hr>
To make the second SVG image scale nicely like the first one, you need to modify the SVG code to include a viewBox attribute, remove the width and height attributes, and add a preserveAspectRatio attribute.

Here's an example of how you can modify the second SVG code:

```xml
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
 "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
 viewBox="0 0 512 512" preserveAspectRatio="xMidYMid meet">

<g transform="translate(0.000000,512.000000) scale(0.100000,-0.100000)"
fill="#000000" stroke="none">
<path d="M2430 5114 c-213 -25 -344 -56 -494 -115 -736 -292 -1173 -1050
-1055 -1833 75 -498 368 -936 804 -1201 173 -104 403 -...Skipped 1 messages
```

In the modified code, the viewBox attribute defines the coordinate system for the SVG image, and the preserveAspectRatio attribute specifies how the SVG image should be scaled and positioned within its container. By using the viewBox attribute and removing the width and height attributes, the SVG image will be able to scale nicely without distortion or pixelation.

<hr>
Here's the part of the file before I modified it according to the instructions above:

```xml
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
 "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
 width="512.000000pt" height="512.000000pt" viewBox="0 0 512.000000 512.000000"
 preserveAspectRatio="xMidYMid meet">

<g transform="translate(0.000000,512.000000) scale(0.100000,-0.100000)"
fill="#000000" stroke="none">
<path d="M2430 5114 c-213 -25 -344 -56 -494 -115 -736 -292 -1173 -1050
```

**Comment by mferrare** [comment]
The male, female and other icons all look slightly different, the male icon is fatter for example.  This should be fixed but I'll open another issue for this.

**Comment by mferrare** [comment]
Issue #28 opened.

**Comment by Sirage-t** [comment]
The issue was indeed because I only accounted for svg in the css file. Now I have added support for img just in case - at least to restrain the size.

Converting png to svg is not always straightforward (ie your converted svg is 24 lines; the one I created is 5).  Also, the reason one appears thicker is because of `stroke-width` which is different from scaling and aspect ratio. 
Anyways I have created proper svg icons to replace existing ones. (I also fixed some padding and alignment issues).

![image](https://user-images.githubusercontent.com/47163720/235336600-b907c994-3bfd-4216-a52c-31a563ed22d6.png)
